app/components/candidate_interface/new_course_choice_needed_banner_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="app-banner app-banner--info" aria-labelledby="info-message">
+  <div class="app-banner__message">
+    <h2 class="govuk-heading-m" id="replace-course-choice-info">
+      <% if @application_form.course_choices_that_need_replacing.count == 1 %>
+          One of your choices is not available anymore.
+      <% else %>
+        Some of your choices are not available anymore.
+      <% end %>
+      <br>
+    <%= govuk_link_to 'Update your course choice now', '#' %>.
+    </h2>
+  </div>
+</div>
```

Comment:
```suggestion
<% if render? %>
<div class="app-banner app-banner--info" aria-labelledby="info-message">
  <div class="app-banner__message">
    <h2 class="govuk-heading-m" id="replace-course-choice-info">
      <% if @application_form.course_choices_that_need_replacing.count == 1 %>
          One of your choices is not available anymore.
      <% else %>
        Some of your choices are not available anymore.
      <% end %>
      <br>
    <%= govuk_link_to 'Update your course choice now', '#' %>.
    </h2>
  </div>
</div>
<% end %>
```

@davidgisbey this is what I meant, instead of calling `render?` on the instance at the call site.